pkgs/development/python-modules/pyairvisual/default.nix

```diff
@@ -43,13 +43,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
-  pytestFlagsArray = [
-    "tests"
-  ];
+  pytestFlagsArray = [ "tests" ];
```

Comment:
Odd, doesn't pytestCheckHook pick up that directory by default?

Reply:
It's more that I forgot zu update it.